File: Desenvolvimento/server/src/ApplicationCore/Entities/FluxoItemSetor.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class FluxoItemSetor : IBaseEntity<long>
    {
        public long Id { get; set; }

        public long SetorId { get; set; }
        public virtual  Setor Setor { get; set; }

        public long FluxoItemId { get; set; }
        public virtual  FluxoItem FluxoItem { get; set; }
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/fluxo-item-setor.ts
import { Setor } from './setor';
import { FluxoItem } from './fluxo-item';
import { BaseModel } from '../base-model';

export class FluxoItemSetor extends BaseModel {
    constructor(
        public id: number,
        public setorId?: number,
        public setor?: Setor,
        public fluxoItemId?: number,
        public fluxoItem?: FluxoItem
    ) { super(); }
}
